panic on RuleList with "from 9.9.9.0/24 to 6.6.6.0/24 tun_id 55 lookup 66"

add rule that breaks RuleList()

`sudo ip rule add from 9.9.9.0/24 to 6.6.6.0/24 lookup 66 tun_id 55`


run code to see panic
```

u@s1:~/junk$ go mod init example
u@s1:~/junk$ go get github.com/vishvananda/netlink@latest
u@s1:~/junk$ go run main.go
panic: runtime error: index out of range [7] with length 4

goroutine 1 [running]:
encoding/binary.littleEndian.Uint64(...)
        /usr/local/go/src/encoding/binary/binary.go:77
github.com/vishvananda/netlink.(*Handle).RuleList(0x56d670, 0x0, 0x401150, 0x0, 0x0, 0xffffffff, 0xc00008e058)
        /home/u/go/pkg/mod/github.com/vishvananda/netlink@v1.1.0/rule_linux.go:207 +0x9f4
github.com/vishvananda/netlink.RuleList(...)
        /home/u/go/pkg/mod/github.com/vishvananda/netlink@v1.1.0/rule_linux.go:160
main.main()
        /home/u/junk/main.go:10 +0x36
exit status 2

```


example code

```
u@s1:~/junk$ cat main.go
package main

import (
        "github.com/vishvananda/netlink"
        "github.com/vishvananda/netlink/nl"
)

func main() {

        netlink.RuleList(nl.FAMILY_ALL)

}
```

--- a/rule_linux.go
+++ b/rule_linux.go
@@ -229,7 +229,7 @@ func (h *Handle) RuleListFiltered(family int, filter *Rule, filterMask uint64) (
 			case nl.FRA_FWMASK:
 				rule.Mask = int(native.Uint32(attrs[j].Value[0:4]))
 			case nl.FRA_TUN_ID:
-				rule.TunID = uint(native.Uint64(attrs[j].Value[0:4]))
+				rule.TunID = uint(native.Uint64(attrs[j].Value[0:8]))
 			case nl.FRA_IIFNAME:
 				rule.IifName = string(attrs[j].Value[:len(attrs[j].Value)-1])
 			case nl.FRA_OIFNAME:

